.travis.yml

```diff
@@ -20,107 +20,31 @@ addons:
 
-# https://docs.travis-ci.com/user/languages/c
-jobs:
-  include:
-    - name: "Coverage gcc7"
-      env:
-        - CXX=g++-7
-        - CC=gcc-7
-        - BUILD=coverage
-        - BUILD_TYPE=Debug
-    - name: "Unit gcc9 (c++2a)"
-      env:
-        - CXX=g++-9
-        - CC=gcc-9
-        - BUILD=unit
-        - BUILD_TYPE=Release
-        - CXXFLAGS="-std=c++2a"
-    - name: "Unit gcc10 (c++17)"
-      env:
-        - CXX=g++-10
-        - CC=gcc-10
-        - BUILD=unit
-        - BUILD_TYPE=Release
-        - CXXFLAGS="-std=c++17 -fconcepts"
-    - name: "Unit gcc10 (c++20)"
-      env:
-        - CXX=g++-10
-        - CC=gcc-10
-        - BUILD=unit
-        - BUILD_TYPE=Release
-    - name: "Unit gcc8"
-      env:
-        - CXX=g++-8
-        - CC=gcc-8
-        - BUILD=unit
-        - BUILD_TYPE=Release
-    - name: "Unit gcc7"
-      env:
-        - CXX=g++-7
-        - CC=gcc-7
-        - BUILD=unit
-        - BUILD_TYPE=Release
-    - name: "Performance gcc7"
-      env:
-        - CXX=g++-7
-        - CC=gcc-7
-        - BUILD=performance
-        - BUILD_TYPE=Release
-    - name: "Header gcc7"
-      env:
-        - CXX=g++-7
-        - CC=gcc-7
-        - BUILD=header
-        - BUILD_TYPE=Release
-    - name: "Snippet gcc7"
-      env:
-        - CXX=g++-7
-        - CC=gcc-7
-        - BUILD=snippet
-        - BUILD_TYPE=Release
-    - name: "Documentation"
-      compiler: 'doxygen'
-      addons:
-        apt:
-          # adds epstopdf, ghostscript, latex, libclang
-          packages: ['texlive-font-utils', 'ghostscript', 'texlive-latex-extra', 'libclang-9-dev', 'libclang-cpp9']
-      env:
-        - BUILD=documentation
-      cache:
-        directories:
-          - /tmp/doxygen-download
-      before_install:
-        - DOXYGEN_VER=1.8.19
-        - DOXYGEN_FOLDER=doxygen-${DOXYGEN_VER}
-        - mkdir -p /tmp/doxygen-download
-        - wget --no-clobber --directory-prefix=/tmp/doxygen-download/ https://sourceforge.net/projects/doxygen/files/rel-${DOXYGEN_VER}/${DOXYGEN_FOLDER}.linux.bin.tar.gz
-        - tar -C /tmp/ -zxvf /tmp/doxygen-download/${DOXYGEN_FOLDER}.linux.bin.tar.gz
-        - PATH=$PATH:/tmp/${DOXYGEN_FOLDER}/bin/
-        - doxygen --version
+env:
+  - BUILD=coverage    CXX=g++-7  CC=gcc-7  BUILD_TYPE=Debug
+  - BUILD=unit        CXX=g++-9  CC=gcc-9  BUILD_TYPE=Release CXXFLAGS="-std=c++2a"
+  - BUILD=unit        CXX=g++-10 CC=gcc-10 BUILD_TYPE=Release CXXFLAGS="-std=c++17 -fconcepts"
+  - BUILD=unit        CXX=g++-10 CC=gcc-10 BUILD_TYPE=Release
+  - BUILD=unit        CXX=g++-8  CC=gcc-8  BUILD_TYPE=Release
+  - BUILD=unit        CXX=g++-7  CC=gcc-7  BUILD_TYPE=Release
+  - BUILD=performance CXX=g++-7  CC=gcc-7  BUILD_TYPE=Release
+  - BUILD=header      CXX=g++-7  CC=gcc-7  BUILD_TYPE=Release
+  - BUILD=snippet     CXX=g++-7  CC=gcc-7  BUILD_TYPE=Release
 
 install:
-  - |
-    HOME_BIN_PATH=`realpath ~/bin`
-    mkdir -p ${HOME_BIN_PATH}
-  - |
-    # install cmake
-    CMAKE_VERSION="3.7.2"
```

Comment:
If we don't fix the cmake version, the log will get to big, we need to fix our cmake script to use the less verbose version.

```cmake
    if (NOT CMAKE_VERSION VERSION_LESS 3.10) # cmake >= 3.10
        gtest_discover_tests(${target} TEST_PREFIX "${test_name}::" PROPERTIES TIMEOUT "30")
    else ()
        add_test (NAME "${test_name}" COMMAND ${target})
    endif ()
```

should be

```cmake
    option (SEQAN3_VERBOSE_TESTS ON)
    if (SEQAN3_TEST_VERBOSE_TESTS AND NOT CMAKE_VERSION VERSION_LESS 3.10) # cmake >= 3.10
        gtest_discover_tests(${target} TEST_PREFIX "${test_name}::" PROPERTIES TIMEOUT "30")
    else ()
        add_test (NAME "${test_name}" COMMAND ${target})
    endif ()
```

Reply:
That means we should run the nightlies with `-DSEQAN3_VERBOSE_TESTS=OFF` ?

Reply:
jop